bundles/org.openhab.binding.hdpowerview/src/main/resources/OH-INF/thing/shade.xml

```diff
@@ -10,6 +10,7 @@
 		</supported-bridge-type-refs>
 		<label>PowerView Shade</label>
 		<description>Hunter Douglas (Luxaflex) PowerView Gen 1/2 Shade</description>
+		<semantic-equipment-tag>WindowCovering</semantic-equipment-tag>
```

Comment:
When should `WindowCovering` vs. `Blinds` be used?

Reply:
WindowCovering is the parent of Blinds and Drapes.

I will add code to dynamically select the one or the other in java code based on the actual shade capabilities.